virttest/utils_net.py

```diff
@@ -1710,6 +1713,10 @@ def create_network_script(iface_name, mac_addr, boot_proto, net_mask,
 
 def ipv6_from_mac_addr(mac_addr):
     """
+    Note:
+    Only support systems which choose EUI-64
+     - RHEL support
```

Comment:
Should be `"Linux"` instead of `"RHEL"`?

Reply:
@luckyh updated. Thanks for review.